packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 14.0.0
+
+* **Breaking Change** [swift] Fixes Flutter Api void return error handling.
+  * This shouldn't be breaking for anyone, but if you were incorrectly getting 
+    success responses, you may now be failing (correctly).
```

Comment:
I believe this is technically breaking, but I'm open to making this 13.1.0 if you think that is more correct.

Reply:
I don't think we need to consider this breaking; if we were just calling the wrong callback method before and now we're not, that's just a bugfix.